client/web/package.json

```diff
@@ -17,12 +17,15 @@
     "isomorphic-unfetch": "^2.0.0",
     "next": "latest",
     "react": "^16.2.0",
+    "react-art": "^16.4.1",
```

Comment:
Why do you require `react-art` here?

Reply:
I'm not sure what actually cause it, but at some point I just got an error saying it was required.